src/Scrolling.md

```diff
@@ -32,10 +32,18 @@ The Window is visible (if enabled) when both coordinates are in the ranges
 WX=0..166, WY=0..143 respectively. Values WX=7, WY=0 place the Window at the
 top left of the screen, completely covering the background.
 
-WX values 0-6 and 166 are unreliable due to hardware bugs. If WX is set
-to 0, the window will "stutter" horizontally when SCX changes.
-(Depending on SCX modulo 8, behavior is a little complicated so you
-should try it yourself.)
+::: warning
+
+WX values 0 and 166 are unreliable due to hardware bugs.
+
+If WX is set to 0, the window will "stutter" horizontally when SCX changes
+(depending on SCX modulo 8, behavior is a little complicated so you
```

Comment:
```suggestion
(depending on SCX modulo 8, detailed behaviour is unknown)
```

Reply:
```suggestion
(depending on SCX modulo 8, exact behavior is a little complicated).
```
It *is* properly understood, I believe.

Reply:
Then why is the reader invited to "try it himself because it's too complicated to explain"?

Reply:
I think because I couldn't find it documented anywhere (guess I could have taken a look at SameBoy's source?), and I thought that it wouldn't be worth explaining in full there. I was probably wrong with that, but then what shall we do?

Reply:
We can link SameBoy source if that's implemented there.

Reply:
I'm not fond of that because AFAIK there is no test ROM for it.

Reply:
If we don't have a source, we can't provide details then we shouldn't mention it at all. Just open an Issue and mark it as research. Makes no sense to add confusion here mentioning stuff in half.

Reply:
Then just
```suggestion
(depending on SCX modulo 8).
```
?

Reply:
Agree!

Reply:
(But still an Issue is worth opening)